wolfcrypt/_build_ffi.py

```diff
@@ -381,12 +381,22 @@ def generate_libwolfssl():
                             RsaKey* key);
     int wc_RsaPublicEncrypt(const byte*, word32, byte*, word32,
                             RsaKey*, WC_RNG*);
-
+    int wc_RsaPublicEncrypt_ex(const byte* in, word32 inLen, byte* out,
```

Comment:
These new additions have `#ifdef` gates in wolfSSL. This is no problem in the case where wolfcrypt-py builds wolfSSL itself (i.e. using _build_wolfssl.py), but it could be problematic when doing feature detection (i.e. using `USE_LOCAL_WOLFSSL`). For example, the PSS stuff is gated on `#ifdef WC_RSA_PSS`. I think we should add some logic to the feature detection stuff in this file to account for that. The *_ex functions also have gates that need to be accounted for.

Reply:
Good point (although the current gating is a bit broken). I'll add it.